.github/workflows/foreman_plugin.yml

```diff
@@ -157,6 +157,11 @@ jobs:
           path: |
             package-lock.json
             ${{ format('{0}/package-lock.json', inputs.plugin) }}
+      - name: Archive all_react_app_exports
```

Comment:
This needs to happen after "Compile webpack" in line 170, otherwise no file is archived.